srml/democracy/src/lib.rs

```diff
@@ -929,7 +929,7 @@ impl<T: Trait> Module<T> {
 			} else {
 				<DispatchQueue<T>>::append_or_insert(
 					now + info.delay,
-					[Some((info.proposal, index))].into_iter()
+					&[Some((info.proposal, index))][..]
```

Comment:
no it will compile with something like:
```
the trait bound `Iter<'_, std::option::Option<(<T as Trait>::Proposal, u32)>>: EncodeLike<std::vec::Vec<std::option::Option<(<T as Trait>::Proposal, u32)>>>` is not satisfied
```

Reply:
hmm at some point we should derive EncodeLike for standard iterator structures.